x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx

```diff
@@ -77,7 +77,7 @@ const getBatchReindexLink = (docLinks: DocLinksStart) => {
   return (
     <FormattedMessage
       id="xpack.upgradeAssistant.esDeprecations.batchReindexingDocsDescription"
-      defaultMessage="To start multiple reindexing tasks in a single request, use the Kibana {docsLink}."
+      defaultMessage="To start upgrading multiple reindexing tasks in a single request, use the Kibana {docsLink} for indices. The {docsLink} does not support data streams."
```

Comment:
I was thinking, perhaps we should also add some clarification to the docs themselves too?

https://www.elastic.co/guide/en/kibana/current/batch-start-resume-reindex.html

`docs/api/upgrade-assistant/batch_reindexing.asciidoc`. WDYT?

Reply:
The asciidoc updates need to be made directly to the pre-9 branches. I'll link back to the original issue for tracking.

Reply:
It's ok to do them here if this whole PR is going to be backported along with it, for simplicity.

But in any case the docs change will have to be done again for 9.0+ in the new docs system.

Reply:
WDYT about using something similar to the kib UI copy: 
"Start or resume upgrading multiple indices in one request. Additionally, reindexing tasks for upgrading indices that are started or resumed via the batch endpoint will be placed on a queue and executed one-by-one. This ensures that minimal cluster resources are consumed over time. Note: this API does not support data streams."

Reply:
Sounds good to me 👍

Reply:
@florent-leborgne the docs update is failing.

Reply:
The "Comment on PR for .asciidoc changes" check doesn't prevent from merging.
Only the "buildkite/docs-build-pr" is needed to merge docs changes